### DISCUSSION TITLE
I Love This Tool! Quick Question

### DISCUSSION THREAD

**Discussion by jakemcgrath** [paukstelis/octoprint_deploy#45] [Q&A] [answered]
First off, huge props this tool is great and is everything I could want. It works well, hats off to you. I was wondering if there was a way, or what file should be edited, to tie an instance's webcam to a USB port as with the printer? I am running into the issue with two cameras having the same default serial number. Can these be tied to a USB port by creating a SYMLINK? What would you recommend to repeat this process the same way the printers are done? Thanks!

**Comment by paukstelis** [comment, marked as answer]
There is already a tool to do this. If you run `sudo octoprint_deploy/octoprint_deploy.sh noserial` it will ignore the serial number for cameras and just use the USB port. If you already have cameras installed, you can join the Discord server and we can walk through the commands to remove the existing cameras.

**Comment by jakemcgrath** [comment]
@paukstelis thank you so much! I really appreciate it. I do have two cameras installed, but I won't be back home until later this evening. If I don't get through it I will take you up on the offer. Totally made my day, let me know if there is anything I can ever help with.
